server/sonar-web/src/main/js/apps/organizations/components/forms/RemoveMemberForm.js

```diff
@@ -21,27 +21,31 @@
 import React from 'react';
 import Modal from 'react-modal';
 import { translate, translateWithParameters } from '../../../../helpers/l10n';
-import type { Member } from '../../../../store/organizationsMembers/actions';
-import type { Organization } from '../../../../store/organizations/duck';
+/*:: import type { Member } from '../../../../store/organizationsMembers/actions'; */
+/*:: import type { Organization } from '../../../../store/organizations/duck'; */
 
+/*::
 type Props = {
   member: Member,
   organization: Organization,
   removeMember: (member: Member) => void
 };
+*/
 
+/*::
 type State = {
   open: boolean
 };
+*/
 
 export default class RemoveMemberForm extends React.PureComponent {
-  props: Props;
+  /*:: props: Props; */
 
-  state: State = {
+  state /*: State */ = {
     open: false
   };
 
-  openForm = (evt: MouseEvent) => {
+  openForm = (evt /*: MouseEvent */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)